src/pages/settings/Profile/ProfilePage.js

```diff
@@ -66,14 +66,12 @@ class ProfilePage extends Component {
     constructor(props) {
         super(props);
 
-        this.defaultAvatar = ReportUtils.getDefaultAvatar(this.props.currentUserPersonalDetails.login);
-
         this.state = {
-            firstName: this.props.currentUserPersonalDetails.firstName,
+            firstName: this.props.currentUserPersonalDetails.firstName || '',
```

Comment:
We can use `lodashGet` for these

Reply:
True I will update!

Reply:
+1

Reply:
What do y'all think about extracting `this.props.currentUserPersonalDetails` in the `constructor` to something like this:

```js
        const currentUserDetails = this.props.currentUserPersonalDetails || {};
        this.state = {
            firstName: currentUserDetails.firstName || '',
            hasFirstNameError: false,
            lastName: currentUserDetails.lastName || '',
            hasLastNameError: false,
            pronouns: currentUserDetails.pronouns || '',
            hasPronounError: false,
            hasSelfSelectedPronouns: !_.isEmpty(currentUserDetails.pronouns) && !currentUserDetails.pronouns.startsWith(CONST.PRONOUNS.PREFIX),
            selectedTimezone: lodashGet(currentUserDetails, 'timezone.selected', CONST.DEFAULT_TIME_ZONE.selected),
            isAutomaticTimezone: lodashGet(currentUserDetails, 'timezone.automatic', CONST.DEFAULT_TIME_ZONE.automatic),
            logins: this.getLogins(props.loginList),
            avatar: {uri: currentUserDetails.avatar || ReportUtils.getDefaultAvatar(currentUserDetails.login)},
            isAvatarChanged: false,
        };
```

Reply:
I like ^ better b/c:
1. We only have to use `lodashGet` where we're accessing nested user detail properties (`timezone.selected` & `timezone.automatic`)
2. It makes some lines a lot shorter, where we were doing `lodashGet(this.props.currentUserPersonalDetails, 'something')` multiple times

Reply:
This can also help us here, where we'd normally want to use `lodashGet` everywhere: https://github.com/Expensify/App/blob/f6736b1a1fffdab3893bfb152dfe0be331d57bb7/src/pages/settings/Profile/ProfilePage.js#L212-L217

Reply:
@Beamanator I am fine with that proposal, nice!

Reply:
Committed!